src/OpenDiffix.Core.Tests/Analyzer.Tests.fs

```diff
@@ -415,6 +415,16 @@ type Tests(db: DBFixture) =
     analyzeQueryUntrusted "SELECT floor(age) from customers" |> ignore
     analyzeQueryUntrusted "SELECT age from customers" |> ignore
 
+  [<Fact>]
+  let ``Detect queries with disallowed bucket functions calls`` () =
+    ensureQueryFailsUntrusted
```

Comment:
is `...Untrusted` necessary here? this should fail in trusted mode as well.

Reply:
exactly, this is why we should `ensureQueryFails` instead, otherwise you're implying the validation is maybe done in untrusted mode only. `trusted` mode is "enough" for the query to fail.